<?xml version="1.0"?>
<launch>
    <arg name="car_frame_id" default="odom"/>
    <arg name="quad_frame_id" default="base_link"/>
    <arg name="camera_frame_id" default="camera_base_link_dummy"/>
    <arg name="scan_topic" default="/scan"/>
    <arg name="planner_config" default="$(find foresight)/param/planner.yaml"/>

    <node pkg="foresight" type="info_planner.py" name="info_planner"
        output="screen">
        <rosparam command="load" file="$(arg planner_config)"/>

        <!-- Frame ids -->
        <param name="map_frame" value="$(arg car_frame_id)"/>
        <param name="quad_frame" value="$(arg quad_frame_id)"/>
        <param name="camera_frame" value="$(arg camera_frame_id)"/>
    </node>

</launch>
